### DISCUSSION TITLE
Svelte/Sveltekit

### DISCUSSION THREAD

**Discussion by GraemeF** [Flagsmith/flagsmith#1267] [Q&A] [answered]
Hello all, wondering if there are any examples using Svelte or ideally Sveltekit? I'm struggling to get anything working as there are different SDKs for client and sever side.

Thanks!

**Comment by dabeeeenster** [comment]
Hi @GraemeF 

We don't currently have examples for Svelte - we don't have experience of the framework. Is there a common approach within Svelte for this sort of thing? What sort of errors are you seeing?

**Comment by kyle-ssg** [comment, marked as answer]
Hey, I’ll take a look at this today.

**Reply by kyle-ssg** [reply]
Hey @GraemeF, https://github.com/Flagsmith/flagsmith-js-client/tree/main/examples/svelte here you go! I think what you were looking for was our isomorphic client which was predominately built for frameworks like this (e.g Next.js) https://docs.flagsmith.com/clients/next-ssr

**Reply by GraemeF** [reply]
Awesome, thank you so much! I'll take a look 👍
